### PR TITLE
Add support for tf 2.2.0 for kubeflow jupyter image

### PR DESCRIPTION
**Pull request by oak-tree**
add tf 2.2.0 script metadata
upgrade `Kubeflow-fairing` because 0.7.1 does not support tf 2.2

### CODE REVIEW

File: components/tensorflow-notebook-image/Dockerfile [hunk 1]
```diff
@@ -113,7 +113,7 @@ RUN pip3 --no-cache-dir install \
     jupyter-console==6.0.0 \
     jupyterlab \
     xgboost \
-    kubeflow-fairing==0.7.1
```
Comment (PatrickXYS) [inline components/tensorflow-notebook-image/Dockerfile:116]: Can you upgrade to latest version 1.0.2?

**Comment by PatrickXYS** [comment]
And sign the CLA to pass check @oak-tree

**Comment by DavidSpek** [comment]
@oak-tree There is now a conflict as the current Dockerfile already has `kubeflow-fairing==1.0.1`.  Could you remove commit https://github.com/kubeflow/kubeflow/pull/5213/commits/2da128518cd1cd36c12c6591c3f74e37a3a0e32e and sign the CLA?

File: components/tensorflow-notebook-image/Dockerfile (hunk 1 above)
Comment (DavidSpek) [reply components/tensorflow-notebook-image/Dockerfile:116]: The dockerfile currently has 1.0.2 in it, so this change is causing a conflict.

**Comment by DavidSpek** [comment]
@oak-tree Are you still interested in merging this PR?

**Comment by DavidSpek** [comment]
@kimwnasptd @thesuperzapper @elikatsis @StefanoFioravanzo Given that there hasn't been a response to this PR for a while, the CLA has not been signed, there is a conflict in the PR and that the notebook images might be refactored soon, can this PR be closed so that it doesn't stay in the backlog indefinitely?

**Pull request closed without merging**